packages/react-components/react-color-picker-preview/library/etc/react-color-picker-preview.api.md

```diff
@@ -53,16 +53,22 @@ export const ColorPicker: ForwardRefComponent<ColorPickerProps>;
 // @public (undocumented)
 export const colorPickerClassNames: SlotClassNames<ColorPickerSlots>;
 
+// @public (undocumented)
+export type ColorPickerOnSelectEventHandler = EventHandler<ColorPickerOnChangeData>;
```

Comment:
Let's avoid exposing such types, it's expected to use `ColorPickerProps['onColorChange']` for example